pages/00-Home.md

```diff
@@ -1,87 +1,67 @@
 
-### RDF and Linked Data
-Hydra is an extension of W3C's Resource Description Framework (RDF). RDF is widely known as the technology which makes the [Semantic Web](https://en.wikipedia.org/wiki/Semantic_Web) possible by representing data as [Linked Data](https://en.wikipedia.org/wiki/Linked_data).
-[RDF is a W3C standard](https://www.w3.org/RDF/) that allows the representation of Knowledge in specific Domain as Knowledge Graphs. Quanta of information are represented as "triples": statements that relate a subject to an object by a predicate. Triples can be stored in different formats. The format used in Hydra and hydrus is [JSON-LD](https://json-ld.org/).
-For a more detailed explanation of triples, you can [read this document](http://www.hydra-cg.com/spec/latest/linked-data-fragments/#interfaces-to-linked-data).
+Diagram shown below should make things more clear. 
 
-### Hydra
-Hydra is a framework that enables REST APIs to be described semantically using RDF. It is based on JSON-LD and proposed as [W3C draft](https://www.hydra-cg.com/spec/latest/core/).
+<img src="https://gsocchrizandr.files.wordpress.com/2017/06/flo1.png"/>
 
+Hydrus supports following features-
+* A client that can understand Hydra vocabulary and interacts with a Hydra supporting server to basic CRUD operations on data.
+* A generic server that can serve required data and metadata(in the form of API documentation) to a client over HTTP.
+* A middleware that allows users to use the client to interact with the server using Natural Language which is processed machine consumable language. __(under development)__.
 
-**hydrus** (lowercase) is the flagship server in the ecosystem. It is a Flask server meant to build and deploy Hydra-based Web APIs in a straightforward and effective way. It uses Docker as its virtualization and isolation technology (see Usage). Other tools, such as dedicated clients for testing, instances running with others frameworks, parsers/translators to other standards, are in the same ecosystem.
+I hope this brief description must have given you a broad Idea of hydrus. If you wish to know more, I would definitely recommend you to try it yourself, Play with it. You can find codebase [here](https://github.com/HTTP-APIs/hydrus). And in case you want to use hydrus, then these [docs](https://hydrus.readthedocs.io/en/latest/hydrus.html#) should be quite helpful. Have fun!
```

Comment:
Again, use third person. Phrases like "I hope"/"I would" works for a blog, not an organisation webpage. Third person voice.